meta-oasis-vm/recipes-kernel/linux/linux-yocto/podman.cfg

```diff
@@ -0,0 +1,85 @@
+..........................................................................
```

Comment:
How does one obtain the needed parameter list?

Reply:
You mean the final kernel config? It is a combination of all fragments.

Reply:
Experimentation and looking around :) Maybe some stuff is not needed.